File: src/main/java/com/github/bjoernpetersen/jmusicbot/user/DuplicateUserException.java
package com.github.bjoernpetersen.jmusicbot.user;

public final class DuplicateUserException extends Exception {

  public DuplicateUserException() {
  }

  public DuplicateUserException(String message) {
    super(message);
  }

  public DuplicateUserException(String message, Throwable cause) {
    super(message, cause);
  }

  public DuplicateUserException(Throwable cause) {
    super(cause);
  }
}

File: src/main/kotlin/com/github/bjoernpetersen/musicbot/spi/auth/DuplicateUserException.kt
package com.github.bjoernpetersen.musicbot.spi.auth

class DuplicateUserException : Exception {
    constructor() : super()
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
    constructor(cause: Throwable) : super(cause)
}
